styles/atomize-d.less

```diff
@@ -5,44 +5,44 @@
 @import "ui-variables";
 
 atom-overlay.autocomplete-plus autocomplete-suggestion-list.autocomplete-suggestion-list.select-list.popover-list {
```

Comment:
have you checked this file yet, if it's even needed anymore?

Reply:
Should probably remove them. I need to first get the language server working. Then, we might have to restore some of the things I removed, or remove other things.